File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/VehicleprofileEngine+CoreDataProperties.m
//
//  VehicleprofileEngine+CoreDataProperties.m
//  WOTData
//
//  Created by Pavel Yeshchyk on 1/16/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

#import "VehicleprofileEngine+CoreDataProperties.h"

@implementation VehicleprofileEngine (CoreDataProperties)

+ (NSFetchRequest<VehicleprofileEngine *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"VehicleprofileEngine"];
}

@dynamic fire_chance;
@dynamic name;
@dynamic power;
@dynamic tag;
@dynamic tier;
@dynamic weight;
@dynamic tankengine;
@dynamic vehicleprofile;

@end

File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/VehicleprofileEngine+CoreDataProperties.swift
//
//  VehicleprofileEngine+CoreDataProperties.swift
//  WOTData
//
//  Created by Pavel Yeshchyk on 1/17/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

import Foundation
import CoreData


extension VehicleprofileEngine {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<VehicleprofileEngine> {
        return NSFetchRequest<VehicleprofileEngine>(entityName: "VehicleprofileEngine")
    }

    @NSManaged public var fire_chance: NSDecimalNumber?
    @NSManaged public var name: String?
    @NSManaged public var power: NSDecimalNumber?
    @NSManaged public var tag: String?
    @NSManaged public var tier: NSDecimalNumber?
    @NSManaged public var weight: NSDecimalNumber?
    @NSManaged public var tankengine: Tankengines?
    @NSManaged public var vehicleprofile: Vehicleprofile?

}
